stencils/KalugaButton+SwiftUI.stencil

```diff
@@ -26,7 +26,7 @@ extension KalugaButton {
                 buttonView
             }
         )
-        .buttonStyle(self.style, isEnabled: self.isEnabled)
+        .buttonStyle(self.style, isEnabled: self.isEnabled_)
```

Comment:
This again? I thought it was fixed using the last Kaluga + KalugaSwiftUI

Reply:
apparently not

Reply:
@cikpis , @Daeda88 any idea on why the issue is back?